lib/install/management/delete.go

```diff
@@ -85,6 +84,10 @@ func (d *Dispatcher) DeleteVCH(conf *metadata.VirtualContainerHostConfigSpec) er
 	if err = d.destroyResourcePoolIfEmpty(conf); err != nil {
 		log.Warnf("VCH resource pool is not removed, %s", err)
 	}
+
+	log.Infoln("Removing volume stores...")
```

Comment:
Yeah, I wasn't sure about this, I'll move it to earlier in the deletion routine.